tc/sgx/trusted_worker_manager/enclave/enclave_data.cpp

```diff
@@ -93,35 +107,55 @@ void EnclaveData::DeserializeSealedData(const std::string& inSerializedEnclaveDa
 
     // Public signing key
     pvalue = json_object_dotget_string(keystore_object, "SigningKey.PublicKey");
+    pvalue_len = strlen(pvalue);
```

Comment:
Do this operation after ValueError check for null pointer. Try to use strnlen_s and max szie  to check will be size of public key,

Reply:
I can add strlen check after value error validation. Currently we don't have max size of public key. We have max size of public enclave data. So, i am not sure how to strlen_s function here.